app/assets/javascripts/statistics.es6

```diff
@@ -0,0 +1,43 @@
+export class Statistics { 
+  constructor(collection_data, work_data) {
```

Comment:
I'm not sure I understand. Can you say more about that?  What exactly are you thinking `tab_content` is?